MobSF/views/api/rest_api.py

```diff
@@ -198,3 +201,16 @@ def api_view_source(request):
     else:
         response = make_api_response({"error": "Missing Parameters"}, 422)
     return response
+
+
+@request_method(['POST'])
+@csrf_exempt
+def api_manifest(request):
+    """
+    """
+    params = ['bin', 'type', 'md5']
+    if set(request.POST) >= set(params):
+        resp = manifest_view.run(request, api=True)

```

Comment:
Handle request based on request type (ie. to support android, iOS and Windows)

Reply:
I don't know about Windows App